static/js/settings_profile_fields.js

```diff
@@ -488,9 +502,10 @@ export function build_page() {
     meta.loaded = true;
 
     $("#admin_profile_fields_table").on("click", ".delete", delete_profile_field);
-    $("#profile-field-settings").on("click", "#add-custom-profile-field-btn", create_profile_field);
+    $("#profile-field-settings").on(
+        "click",
+        "#add-custom-profile-field-btn",
+        open_custom_profile_field_form_modal,
+    );
```

Comment:
You can just do `$("#add-custom-profile-field-btn").on("click", open_custom_profile_field_form_modal);`